packages/tools/pluggable-widgets-tools/scripts/e2e.js

```diff
@@ -31,15 +31,21 @@ async function main() {
 
     const packageConf = JSON.parse(await readFile("package.json"));
     const widgetVersion = packageConf?.version;
-    const widgetMpk = ls(`dist/${widgetVersion}/*.mpk`).length;
 
-    if (!widgetMpk) {
-        throw new Error("No widgets founds in dist folder. Please execute `npm run release` before start e2e tests.");
-    }
+    // TODO [https://mendix.atlassian.net/browse/WT-3111]: Use this flag via `"test:e2e": "pluggable-widgets-tools test:e2e:web --skip-mpk-check"` in atlas package.json
+    // When they released a working 9.x docker image
+    if (!process.argv.includes("--skip-mpk-check")) {
```

Comment:
I doubt about the option name. Basically we want to prevent copying over the mpk to test project folder. Maybe `--no-mpk` or `--without-mpk` is better.

Reply:
Not just preventing the copying over, also "checking" if the mpk is there. But anyhow we are making this check because the project we are running is not for testing "widget" but testing something else. From here i would change it to `skip-widget-mpk-check`

Reply:
Basically we skip updating the test project: `--skip-test-project-updating`.

Reply:
Btw, I don't see this option being used anywhere at the moment.

Reply:
--no-widget-update ?